packages/did-comm/src/didcomm.ts

```diff
@@ -141,6 +142,16 @@ export interface IPackDIDCommMessageArgs {
   options?: IDIDCommOptions
 }
 
+/**
+ * The config for the {@link DIDComm} DIDComm plugin.
+ *
+ * @beta This API may change without a BREAKING CHANGE notice.
+ */
+export interface DIDCOmmConfig<T extends IDIDCommTransport = DIDCommHttpTransport> {
+  transports?: T[]
+  isMediateDefaultGrantAll?: boolean
```

Comment:
@radleylewis You know this already, but I'll jot it down here. Move this parameter to the `CoordinateMediationMediatorMessageHandler` constructor. You'll need to put the `grantOrDenyMediation` inside the `CoordinateMediationMediatorMessageHandler` class, so it has access to the argument.

Reply:
Sounds good.

Reply:
This has  been shifted to the `MediationManagerPlugin`.